engine/src/main/java/org/terasology/logic/characters/CharacterSystem.java

```diff
@@ -62,7 +64,10 @@
 import org.terasology.world.block.BlockComponent;
 import org.terasology.world.block.regions.ActAsBlockComponent;
 
+import java.util.Optional;
+
 /**
+ *
  */
```

Comment:
are you planning on added some javadocs to this class?

Reply:
eventually yes - this is just an auto-formatting change. The `CharacterSystem` does a lot of stuff right now, maybe it can be slimmed down to have a better idea how each part is working...